static.json

```diff
@@ -0,0 +1,12 @@
+{
+  "root": "dist",
+  "clean_urls": true,
+  "routes": {
+    "/**": "index.html"
+  },
+  "proxies": {
+    "/api/": {
+      "origin": "https://mitra-social-backend.herokuapp.com"
```

Comment:
I don't think in general if we have any hardcoded URLs in the code... maye we can use an env variable here somehow as well?